### DISCUSSION TITLE
Proposed Documentation: Loading Builder

### DISCUSSION THREAD

**Discussion by joshkautz** [flame-engine/flame#873] [Ideas]
Hey there. Been lurking around this project for several months as I've been working on a game. I spent some time figuring out how to use **loadingBuilder** in game widgets; I was thinking that this could be a good thing to write some documentation on for handling most use-cases of loading screens and splash screens. I know I went down a little rabbit hole looking at the FlameSplashScreen widget and it was ultimately not quite what I ended up needing.

Would folks be on board if I decided to write out a little doc about using the loadingBuilder for game widgets?

**Comment by spydon** [comment]
Deleted my other comments, because apparently I can't read. :sweat_smile: 
We would be super happy if you would write docs for it!
Maybe add a section about it in this file: https://github.com/flame-engine/flame/blob/main/doc/game.md

**Reply by spydon** [reply]
If you want to you could also add an example for it in here: https://github.com/flame-engine/flame/tree/main/examples
